### DISCUSSION TITLE
Specialty Quality of Life change

### DISCUSSION THREAD

**Discussion by SeaSaltSong** [JohanFalt/Foundry_WoD20#397] [Ideas]
I love that you made this, I'm running a game and having the sheets from different systems available for NPCs is amazing.

One thing I've noticed is that specialties are on a separate tab entirely rather than just displayed next to the attribute/ability. My players are rather forgetful and will likely forget about their specialties if it's in a separate section so I was hoping there was some way to fix that.

I get this is a very new project and some of the aesthetic tidying up has not been done yet, thanks for making a whole WoD system rather than one for a specific game.

**Comment by JohanFalt** [comment]
This has been done in v2.4

**Comment by JohanFalt** [comment]
Graphics in v3 should have fixed this.
